library/src/main/java/com/novoda/downloadmanager/lib/NotificationDisplayer.java

```diff
@@ -114,17 +114,15 @@ private void buildActionIntents(String tag, int type, Collection<DownloadBatch>
             hideIntent.putExtra(DownloadReceiver.EXTRA_BATCH_ID, batchId);
             builder.setDeleteIntent(PendingIntent.getBroadcast(context, 0, hideIntent, 0));
 
-            if (type == DownloadNotifier.TYPE_SUCCESS) {
```

Comment:
so this means that now we do this for any `TYPE`. Is that what we want? OR should you extend the if condition instead of removing it?